docs/reference/commandline/pull.md

```diff
@@ -233,18 +234,16 @@ Killing the `docker pull` process, for example by pressing `CTRL-c` while it is
 running in a terminal, will terminate the pull operation.
 
 ```console
-$ docker pull fedora
+$ docker pull ubuntu
 
 Using default tag: latest
-latest: Pulling from library/fedora
+latest: Pulling from library/ubuntu
 a3ed95caeb02: Pulling fs layer
 236608c7b546: Pulling fs layer
 ^C
 ```
 
-> **Note**
->
-> The Engine terminates a pull operation when the connection between the Docker
-> Engine daemon and the Docker Engine client initiating the pull is lost. If the
-> connection with the Engine daemon is lost for other reasons than a manual
-> interaction, the pull is also aborted.
+The Engine terminates a pull operation when the connection between the Docker
+Engine daemon and the Docker Engine client initiating the pull is lost. If the
+connection with the Engine daemon is lost for other reasons than a manual
+interaction, the pull is also aborted.
```

Comment:
The formulation is bit unclear. I would formulate it like the pull is aborted when: 1) connection between daemon and client is cut or lost or the operation is manually terminated.

```suggestion
The Engine terminates a pull operation when the connection between the daemon and the client (initiating the pull) is cut or lost for any reason or the command is manually terminated. 
```